scripts/macro-usage-report.ts

```diff
@@ -115,15 +115,18 @@ async function getDeprecatedMacros() {
 
 function formatCell(files: string[], limit = 2): string {
   if (files.length === 0) {
-    return "";
+    return "0";
```

Comment:
It felt like the empty cell makes it easier to spot where occurrences are, because essentially our brain doesn't need to parse the 0.

Reply:
Could we make it a `-` perhaps?

Reply:
Yes, that's a good compromise.

```suggestion
    return "-";
```